network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx

```diff
@@ -0,0 +1,246 @@
+
+### Importing your data
+
+Now that we've provisioned a new tenant, we can import our data into it. We need to copy our old tables' data to the new tenant schema.
+
+An important note on this is that there are some tables that we don't need to copy. These tables are:
+
+- **_gorp_migrations_**
+- **_auth_migrations_**
+- **_countries_**
+- **_organizations_**
+
+To import your data it's important to start with the following tables but we need to delete the pre-existing entries from the provisioned tenant, before importing it. These tables are:
+
+- **_auth_users_**
+- **_wallets_assets_**
+- **_assets_**
+- **_wallets_**
+
+The following SQL script deletes the entries on those tables:
+
+<CodeExample>
+
+```sql
+DELETE FROM sdp_<tenant name>.auth_users;
+DELETE FROM sdp_<tenant name>.wallets_assets;
+DELETE FROM sdp_<tenant name>.assets;
+DELETE FROM sdp_<tenant name>.wallets;
+```
+
+</CodeExample>
+
+After this step, you can import the data for those tables. Remember importing them in the opposite order they were deleted.
+
+Next, we can import the tables left. We recommend following the table order described here because foreign keys references:
+
+- **_receivers_**
+- **_receiver_wallets_**
+- **_receiver_verifications_**
+- **_disbursements_**
+- **_payments_**
+- **_messages_**
+- **_auth_user_mfa_codes_**
+- **_Auth_user_password_reset_**
+
+_**Note**: some columns like **status** and **types** have their enum type, so you probably will need to cast values to the type (i.e. **status::sdp_bluecorp.payment_status**)_
+
+With this, we complete importing the SDP data but still need to import the Transaction Submitter Service data. To that, we will import our data to the tss schema tables. But before importing the submitter_transactions table we need to get the tenant ID generated for our provisioned tenant:
+
+<CodeExample>
+
+```sql
+SELECT id, name FROM public.tenants;
+```
+
+</CodeExample>
+
+This query retrieves the ID for the tenant.
+
+- **_channel_accounts_** will be imported to **_tss.channel_accounts_**
+- **_submitter_transactions_** will be imported to **_tss.submitter_transactions_**. Note that 2 new columns were added for this table on this version **_tenant_id_** and **_distribution_account_**, for the **_tenant_id_** column we will add the ID we got before, and for the **_distribution_account_** we set the public key of the distribution account.
+- **_tss.vault_**  this table is used for storing encrypted distribution account private keys when `DISTRIBUTION_SIGNER_TYPE=DISTRIBUTION_ACCOUNT_DB`
+
+Now, the migration from the non-tenant version to the tenant version is complete. You can log in to your SDP
+
+### [Optional] Drop old tables
+
+In case you applied these changes to the same database you were using before, you may now drop the SDP tables from the public schema since your tenant now has its own schema. Please, be careful running these commands since it can result in permanent data loss.
+
+<CodeExample>
+
+```sql
```

Comment:
I believe DDL in postgres doesn't cause implicit commit based on this reading. I think if that's case, we should wrap this in a transaction i.e. `BEGIN TRANSACTION; DROP.... COMMIT;` 
https://wiki.postgresql.org/wiki/Transactional_DDL_in_PostgreSQL:_A_Competitive_Analysis